modules/scripts/startup-script/variables.tf

```
@@ -262,3 +262,9 @@ variable "http_no_proxy" {
   default     = ".google.com,.googleapis.com,metadata.google.internal,localhost,127.0.0.1"
   nullable    = false
 }
+
+variable "install_rdma" {
+  description = "If set to true will install and enable Cloud RDMA drivers (supported on Rocky 8 images)."
```

nit: Just for clarity, separate into two sentences:

```suggestion
  description = "If true, will install and enable Cloud RDMA drivers.  Currently only supported on Rocky 8 images."
```